src/framework/components/camera/component.js

```diff
@@ -29,157 +32,204 @@ import { Component } from '../component.js';
  * @example
  * // Update a property on a camera component
  * entity.camera.nearClip = 2;
- * @property {number} projection The type of projection used to render the camera. Can be:
+ * @property {number} projection The type of projection used to render the camera.

```

Comment:
Is it possible to change pc.PROJECTION_ enum to use strings instead of numbers? Much nicer for debugging!

Reply:
Yes! I like this idea. But I'd rather not make this part of this PR. An issue requesting this is fine but begs the question about doing a proper audit of _all_ engine constants and make their type consistent across the whole engine.

Reply:
great, happy to know we can do this at some point.